test/geometries/boost_array_as_point.cpp

```diff
@@ -31,6 +35,12 @@ int test_main(int, char* [])
     std::clog << bg::distance(p1, p2) << std::endl;
     std::clog << bg::distance(p2, p3) << std::endl;
     std::clog << bg::distance(p3, p4) << std::endl;
+
+#ifndef BOOST_NO_CXX11_HDR_ARRAY 
+    std::array<double, 3> p5{13,14,15};
+    std::clog << bg::distance(p4, p5) << std::endl;
+#endif //BOOST_NO_CXX11_HDR_ARRAY 
```

Comment:
At first I thought this test is incomplete too, but now I think it's not too bad. Since this test merely tests if the new point type is recognized by using the relatively simple `bg::distance()` algorithm and hope this test compiles. So the only thing I'd maybe add to these adaptor tests would be a check for the distance results in order to test if the point layout has been recognized correctly. But that's probably a question of taste.